### ISSUE TITLE
OBIS portal search doesn't show the network

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#4350]
**OBIS portal search doesn&#39;t show the network**

I think we used to have a special search result for OBIS to the network page.

-----

User: [See in registry](https://www.gbif.org/api/feedback/user/f62268c19e422f613002f0a5889a0105:c584b9b7132eef39ba94f2e390acc7f57afad651b56a86ad37905f107dae33cac670d86af584586f5add1ba64a53593b8a4a281f2be95c3a4b7e37b52b0aff3ac120b0a0b98e486017b55a53dc19cc5c3bf6f49959fb5adc09e532de3e90955e09bb58fe248259c0cf9ae920312e03ddc49365785e2e9c380ef494ad501c08b0cd388e236561614fa3143af982be9a83) - [Send email](https://www.gbif.org/api/feedback/user/mailto/f62268c19e422f613002f0a5889a0105:c584b9b7132eef39ba94f2e390acc7f57afad651b56a86ad37905f107dae33cac670d86af584586f5add1ba64a53593b8a4a281f2be95c3a4b7e37b52b0aff3ac120b0a0b98e486017b55a53dc19cc5c3bf6f49959fb5adc09e532de3e90955e09bb58fe248259c0cf9ae920312e03ddc49365785e2e9c380ef494ad501c08b0cd388e236561614fa3143af982be9a83)
System: Firefox Mobile 105.0.0 / Android 0.0.0
Referer: https://www.gbif.org/search?q=Obis
Window size: width 360 - height 568
[API log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2022-10-17T17:29:41.694Z',to:'2022-10-17T17:35:41.694Z'))&_a=(columns:!(_source),filters:!(),index:'3390a910-fcda-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
[Site log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2022-10-17T17:29:41.694Z',to:'2022-10-17T17:35:41.694Z'))&_a=(columns:!(_source),filters:!(),index:'5c73f360-fce3-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
System health at time of feedback: OPERATIONAL

**Comment by MortenHofft** [comment]
It relies on keywords to be filled in Contentful and to have the correct casing which is lower case. Perhaps we used to have a normalizer that lowercased it for search.

The way it works is that it does a term query for the keyword lower cased. And results that use that exact keyword is bumped to the top of the page (normally we show resource results last). Since we cannot compare scores against indices, then this is the way we know that a resource result is worth putting first.

@fmendezh perhaps we could add a lowercase normalizer to the keywords to ease searching. I could also use a more loose matching, but the deal with content writers is that we do not use fuzzy matching for this. It is exact keywords only.

I've changed the keyword in the article to ensure it goes to the top.
